src/wazuh_modules/wm_database.c

```diff
@@ -848,7 +848,7 @@ int wm_sync_file(const char *dirname, const char *fname) {
         if (id_agent) {
             switch (wm_extract_agent(fname, name, addr, &is_registry)) {
             case 0:
-                if ((id_agent = wdb_find_agent(name, addr)) < 0) {
+                if ((id_agent = wdb_find_agent(name, NULL)) < 0) {
```

Comment:
Is there any particular reason why we need this change? The SQL_FIND_AGENT query seems to work even when the address is '**_any'_**